panel/chat/message.py

```diff
@@ -437,7 +436,7 @@ def _avatar_lookup(self, user: str) -> Avatar:
         }
         # now lookup the avatar
         return updated_avatars.get(alpha_numeric_key, self.avatar).format(
-            dist_path=self._dist_path
```

Comment:
Sadly even this is likely not enough, it has to be recomputed in `_get_model` somehow. I had a look but didn't see an obvious/easy way to do it.

Reply:
I'm confused because it seems to work:

```python
import panel as pn

pn.extension()

area = pn.chat.ChatInterface(user="wolfram")
area.servable()
```

<img width="173" alt="image" src="https://github.com/holoviz/panel/assets/15331990/76f0caa6-cae1-4dde-b0c2-a3597609b289">

Reply:
It will work in most basic cases but will break when using prefixed paths, nested routes and a few other conditions.